app/assets/javascripts/spotlight/users.js

```diff
@@ -27,13 +27,15 @@
     }
 
     function clear_errors(element) {
-      element.find('.has-error').removeClass('has-error');
-      element.find('.help-block').remove(); // Remove the error messages
+      element.find('.has-error')
```

Comment:
It is because `removeClass` is going to return the element that the class was removed on (which is the `.form-group`) and the `.help-block` is a child of that `.form-group`.